scss/prototype/_display.scss

```diff
@@ -0,0 +1,50 @@
+// Foundation for Sites by ZURB
+// foundation.zurb.com
+// Licensed under MIT Open Source
+
+////
+/// @group prototype-display
+////
+
+/// Responsive breakpoints for display, defaulted to global-prototype-responsive-breakpoints.
+/// @type Boolean
+$prototype-display-responsive-breakpoints: $global-prototype-responsive-breakpoints !default;
+
+@mixin foundation-prototype-display {
+  @each $display in (none, inline, inline-block, block, table, table-cell) {
```

Comment:
@brettsmason need your advice here 
I am thinking of adding a map variable like `$display-map`
Here's what i am looking for, 

```scss
$display-map: (
  none, inline, inline-block, block, table, table-cell
)!default;

@each $display in $display-map {
  /* All my styling here */
}
```